x-pack/plugins/reporting/server/lib/screenshots/observable.ts

```diff
@@ -86,13 +86,9 @@ export function screenshotsObservableFactory(
                 );
               }),
               mergeMap(() => getNumberOfItems(captureConfig, driver, layout, logger)),
-              mergeMap(async (itemsCount) => {
-                const viewport = layout.getViewport(itemsCount) || getDefaultViewPort();
```

Comment:
We can't update the viewport here - it has to be done right before capturing the screenshot since it includes info that comes from `getElementPositionAndAttributes`.